game-core/src/test/java/games/strategy/engine/message/unifiedmessenger/RemoteMessengerTest.java

```diff
@@ -211,11 +208,9 @@ public void testShutDownClient() throws Exception {
       final UnifiedMessenger serverUnifiedMessenger = new UnifiedMessenger(server);
       final RemoteMessenger clientRemoteMessenger = new RemoteMessenger(new UnifiedMessenger(client));
       clientRemoteMessenger.registerRemote(new TestRemote(), test);
-      serverUnifiedMessenger.getHub().waitForNodesToImplement(test.getName());
-      assertTrue(serverUnifiedMessenger.getHub().hasImplementors(test.getName()));
```

Comment:
These assertions are no longer necessary because Awaitility will throw an exception if the condition is not met within the specified timeout.